pkg/nuctl/test/function_test.go

```diff
@@ -225,6 +225,22 @@ func (suite *functionDeployTestSuite) TestDeployFromFunctionConfig() {
 	// make sure to clean up after the test
 	defer suite.dockerClient.RemoveImage(imageName) // nolint: errcheck
 
+	// clear output buffer from last invocation
+	suite.outputBuffer.Reset()
+
+	// export the function
+	err = suite.RetryExecuteNuctlUntilSuccessful([]string{"get", "fu", functionName}, map[string]string{
+		"output": "yaml",
+	}, false)
+	suite.Require().NoError(err)
+
+	deployedFunctionConfig := functionconfig.Config{}
+	err = yaml.Unmarshal(suite.outputBuffer.Bytes(), &deployedFunctionConfig)
+	suite.Require().NoError(err)
+
+	suite.Require().Equal(1, len(functionconfig.GetTriggersByKind(deployedFunctionConfig.Spec.Triggers, "http")))
```

Comment:
Add comment explaining the function already has 1 http trigger, api and here you verify the default HTTP trigger wasn't added